inc/saml/namespace.php

```diff
@@ -76,4 +77,36 @@ function remove_plugin_admin_ui() {
 	remove_action( 'admin_init', 'HumanMade\\SimpleSaml\\Admin\\settings_fields' );
 	remove_action( 'wpmu_options', 'HumanMade\\SimpleSaml\\Admin\\network_settings_fields' );
 	remove_action( 'update_wpmu_options', 'HumanMade\\SimpleSaml\\Admin\\save_network_settings_fields' );
+
+	// Remove built-in login form UI.
+	remove_action( 'login_message', 'HumanMade\\SimpleSaml\\login_form_link' );
+}
+
+/**
+ * Show SSO login link in login form
+ *
+ * @action login_form
+ */
+function render_login_link() {
+	/**
+	 * Filters whether we should show the SSO login link in login form
+	 *
+	 * @return bool  Forces SSO authentication if true, defaults to True
```

Comment:
Typically I've documented these as parameters with it being implicit that the 1st parameter should be returned. That's what the doc blocks in core do so presumably that's required for the WP documentation generator.

```suggestion
	 * @param bool $force_sso Forces SSO authentication if true, defaults to True.
```

Reply:
These are actually all copied from WP Simple SAML; should we fix and copy back?

Reply:
Yeah I think so.